Source/Core/Common/Network.cpp

```diff
@@ -187,6 +297,258 @@ u16 ComputeNetworkChecksum(const void* data, u16 length, u32 initial_value)
   return ~static_cast<u16>(checksum);
 }
 
+// Compute the TCP network checksum with a fake header
+u16 ComputeTCPNetworkChecksum(const IPAddress& from, const IPAddress& to, const void* data,
+                              u16 length, u8 protocol)
+{
+  // Compute the TCP checksum with its pseudo header
```

Comment:
I don't think I saw it last time so its probably an artifact after you rebased. Both comments state the same thing. You might want to remove the one inside the function.